sdk/identity/azure-identity/src/main/java/com/azure/identity/IdentityLogOptions.java

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.identity;
+
+import com.azure.core.annotation.Fluent;
+
+/**
+ * Represents the Identity Log options used to configure logging on the Token Credentials.
+ */
+@Fluent
+public final class IdentityLogOptions {
+    private boolean allowLoggingAccountIdentifiers;
+
+    /**
+     * Sets the flag indicating whether client side account identifier logs should be logged or not.
+     * The default value is false.
+     * <p>
+     * The Account Identifier logs can contain sensitive information and should be enabled on protected machines only.
+     * <p>
+     *
+     * @param allowLoggingAccountIdentifiers The flag indicating if client side account identifier logging should be
+     * enabled or not.
+     * @return The updated IdentityClientOptions object.
+     */
+    public IdentityLogOptions setAllowLoggingAccountIdentifiers(boolean allowLoggingAccountIdentifiers) {
```

Comment:
Use same method name suffix as the getter method `setLoggingAccountIdentifiersAllowed`